README.md

```diff
@@ -156,26 +153,42 @@ adaptive_lighting:
 
 `adaptive_lighting.apply` applies Adaptive Lighting settings to lights on demand.
 
-| Service data attribute | Required | Description                                                                                  |
-| ---------------------- | -------- | -------------------------------------------------------------------------------------------- |
-| `entity_id`            | ✅        | The `entity_id` of the switch with the settings to apply.                                    |
-| `lights`               | ❌        | A light (or list of lights) to apply the settings to.                                        |
-| `transition`           | ❌        | The number of seconds for the transition.                                                    |
-| `adapt_brightness`     | ❌        | Whether to change the brightness of the light or not.                                        |
-| `adapt_color`          | ❌        | Whether to adapt the color on supporting lights.                                             |
-| `prefer_rgb_color`     | ❌        | Whether to prefer RGB color adjustment over of native light color temperature when possible. |
-| `turn_on_lights`       | ❌        | Whether to turn on lights that are currently off.                                            |
+<!-- START_CODE -->
+<!-- from homeassistant.components.adaptive_lighting import _docs_helpers -->
+<!-- print(_docs_helpers.generate_apply_markdown_table()) -->
+<!-- END_CODE -->
+
+<!-- START_OUTPUT -->
+<!-- THIS CONTENT IS AUTOMATICALLY GENERATED -->
+| Service data attribute   | Description                                                                    | Required   | Type                 |
+|:-------------------------|:-------------------------------------------------------------------------------|:-----------|:---------------------|
+| `entity_id`              | Entity ID of the switch. 📝                                                     | ✅          | list of `entity_id`s |
+| `lights`                 | List of light entities to be controlled by Adaptive Lighting (may be empty). 🌟 | ❌          | list of `entity_id`s |
```

Comment:
I notice this is incorrect because 'lights' here doesn't mean the same as in the config.

Will fix this later.